### PR TITLE
Draft email campaign machines

### PR DESCRIPTION
**Pull request by bradwright**
Add machines for draft-email-campaign-frontend VDC.

### CODE REVIEW

File: hieradata/integration.yaml [hunk 1]
```diff
@@ -170,6 +170,9 @@ govuk::node::s_cache::protect_cache_servers: true
 govuk::node::s_frontend_lb::calculators_frontend_servers:
   - 'calculators-frontend-1.frontend'
   - 'calculators-frontend-2.frontend'
+govuk::node::s_frontend_lb::draft_email_campaign_frontend_servers:
+  - 'draft-email-campaign-frontend-1.frontend'
+  - 'draft-email-campaign-frontend-2.frontend'
```
Comment (alexmuller) [inline hieradata/integration.yaml]: This parameter will need to be added to the `govuk::node::s_frontend_lb` class.

Comment (bradwright) [inline hieradata/integration.yaml]: @alexmuller fixed and force pushed.

File: hieradata/common.yaml [hunk 2]
```diff
@@ -606,6 +606,10 @@ hosts::production::frontend::hosts:
     ip: '10.1.2.2'
   frontend-2:
     ip: '10.1.2.3'
+  draft-email-campaign-frontend-1:
```
Comment (mattbostock) [inline hieradata/common.yaml]: You'll need to define IP addresses for each environment if you wish to add these machines to each environment.

Comment (bradwright) [inline hieradata/common.yaml]: @mattbostock fixed and added to PR.

**Comment by mattbostock** [comment]
:+1: Merge when ready (the machines will need provisioning first (done in https://github.gds/gds/govuk-provisioning/pull/516/files))

**Comment by alexmuller** [comment]
> the machines will need provisioning first

Small nitpick: I think this can (and in fact needs to) be merged and deployed before the machines are provisioned.

**Comment by mattbostock** [comment]
Sorry, you're right @alexmuller :blush:

**Comment by bradwright** [comment]
So... should I merge this?

**Comment by alexmuller** [comment]
Yep :100:

**Comment by bradwright** [comment]
Thanks @alexmuller

**Pull request merged**